types/screenshot-desktop/index.d.ts

```diff
@@ -0,0 +1,53 @@
+// Type definitions for screenshot-desktop 1.12
+// Project: https://github.com/baz/foo (Does not have to be to GitHub, but prefer linking to a source code repository rather than to a project website.)
+// Definitions by: Usama Ahsan <https://github.com/usama8800>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="../node" />
+
+/*~ Note that ES6 modules cannot directly export callable functions.
+ *~ This file should be imported using the CommonJS-style:
+ *~   import x = require('someLibrary');
+ *~
+ *~ Refer to the documentation to understand common
+ *~ workarounds for this limitation of ES6 modules.
+ */
+
+/*~ This declaration specifies that the function
+ *~ is the exported object from the file
+ */
+export = ScreenshotDesktop;
+
+/*~ This example shows how to have multiple overloads for your function */
+declare function ScreenshotDesktop(options?: { format?: ScreenshotDesktop.ImageFormat, screen?: ScreenshotDesktop.DisplayID }): Promise<Buffer>;
```

Comment:
```ts
declare function ScreenshotDesktop(options?: { format?: ScreenshotDesktop.ImageFormat, screen?: ScreenshotDesktop.DisplayID }): Promise<Buffer>;
```
I'd suggest to use JS conventions, where CamelCase is for class or constructor function, say `screenshot` (like in the Readme) or `screenshotDesktop`